docs/integrations/aws-lambda.md

```diff
@@ -0,0 +1,135 @@
+---
+id: integration-aws-lambda
+title: Integrando Pix com AWS Lambda (serverless)
+sidebar_label: AWS Lambda
+---
+
+### Configurando a AWS CLI
+
+Precisamos ter a `aws` CLI configurada também. É necessário, pois ela quem irá controlar toda
+a questão de acesso para a sua conta e o script que vai fazer o _deploy_ da sua aplicação, via CDK.
+
+Neste [link](https://aws.amazon.com/cli/), contém a instalação para a CLI de acordo com o seu
+sistema operacional.
+
+Depois disso será necessário que você configure a CLI a partir do comando: `aws configure`. Ele irá
+pedir alguns valores como: `AWS Access Key ID`, `AWS Secret Access Key`, etc. Para entender como de
```

Comment:
are these lines right?

Reply:
related to break line behavior

Reply:
yes, it just to avoid plus than 80 characters per line

on markdown engine, it'll be rendered as the same _paragraph_